rustls/src/crypto/mod.rs

```diff
@@ -582,9 +582,9 @@ pub trait ActiveKeyExchange: Send + Sync {
     fn complete_for_tls_version(
         self: Box<Self>,
         peer_pub_key: &[u8],
-        tls_version: &SupportedProtocolVersion,
+        tls_version: ProtocolVersion,
     ) -> Result<SharedSecret, Error> {
-        if tls_version.version() != ProtocolVersion::TLSv1_2 {
+        if tls_version != ProtocolVersion::TLSv1_2 {
```

Comment:
Note this is a wider type, so `complete_for_tls_version(..., ProtocolVersion::TLSv1_0)` is admitted now. So this branch is probably more accurately `if tls_version >= ProtocolVersion::TLSv1_3` perhaps.